### PR TITLE
[MIG][10] web_widget_many2many_tags_multi_selection

### PR DESCRIPTION
**Pull request by fmdl**

### CODE REVIEW

File: web_widget_many2many_tags_multi_selection/views/web_widget_many2many_tags_multi_selection.xml [hunk 1]
```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0" encoding="utf-8"?>
-<openerp>
+<odoo>
     <data>
```
Comment (pedrobaeza) [inline web_widget_many2many_tags_multi_selection/views/web_widget_many2many_tags_multi_selection.xml]: You can omit data tag if you want

**Review by pedrobaeza — APPROVED** [review]
Squashing the commits

File: web_widget_many2many_tags_multi_selection/__manifest__.py [hunk 2]
```diff
@@ -21,7 +21,7 @@
 
 {
     'name': 'Tags multiple selection',
-    'version': '9.0.1.1.0',
+    'version': '10.0.1.1.0',
```
Comment (pedrobaeza) [inline web_widget_many2many_tags_multi_selection/__manifest__.py]: Sorry, last change: please put 10.0.1.0.0

**Review by tarteo — CHANGES_REQUESTED** [review]
In the README it says:
> Installation
> 
> It was tested on Odoo 8.0 branch.

Can you remove or change this please?

Tested it on Runbot 👍

**Review by tarteo — APPROVED** [review]

**Comment by jesusVMayor** [comment]
Functional review :thumbsup:
@Yajo

**Review by jesusVMayor — APPROVED** [review]

**Pull request merged**